docs/docs/installation/configuring-superset.mdx

```diff
@@ -106,7 +106,7 @@ Superset supports the following database engines/versions:
 | Database Engine                                           | Supported Versions                 |
 | --------------------------------------------------------- | ---------------------------------- |
 | [PostgreSQL](https://www.postgresql.org/)                 | 10.X, 11.X, 12.X, 13.X, 14.X, 15.X |
-| [MySQL](https://www.mysql.com/)                           | 5.X                                |
+| [MySQL](https://www.mysql.com/)                           | 5.7, 8.X                           |
```

Comment:
It seemed prudent to list the supported versions of MySQL, i.e., those which are the [currently supported](https://en.wikipedia.org/wiki/MySQL#Release_history).